x/ccv/provider/keeper/proposal.go

```diff
@@ -188,6 +188,12 @@ func (k Keeper) StopConsumerChain(ctx sdk.Context, chainID string, closeChan boo
 
 		// delete VSC send timestamps
 		k.DeleteVscSendTimestampsForConsumer(ctx, chainID)
+
+		// delete consumer commission rate
```

Comment:
Why is this called inside `if channelID, found := k.GetChainToChannel(ctx, chainID); found {` and not outside at the same location where `k.DeleteKeyAssignments(ctx, chainID)` is located?